.circleci/config.yml

```diff
@@ -9,114 +9,115 @@ jobs:
     steps:
       - checkout
       - run:
-          name: install black
-          command: |
-            sudo pip install black
+	  name: install black
+	  command: |
+	    sudo pip install black
       - run:
-          name: run black
-          command: |
-            black . --check
+	  name: run black
+	  command: |
+	    black . --check
   deploy_to_playground:
     docker:
       - image: circleci/python:3.6-stretch
     working_directory: ~/dash-sample-apps
     steps:
       - checkout
       - run:
-          name: Add plotly remote
-          command: git remote add playground https://dash-playground.plotly.host/GIT/$CIRCLE_BRANCH
+	  name: Add plotly remote
+	  command: git remote add playground https://dash-playground.plotly.host/GIT/$CIRCLE_BRANCH
       - run:
-          name: Create helper-script
-          command: printf '#!/bin/bash\necho username=$PLAYGROUND_DEPLOY_USERNAME\necho password=$PLAYGROUND_DEPLOY_PASSWORD' >> /home/circleci/helper-script.sh
+	  name: Create helper-script
+	  command: printf '#!/bin/bash\necho username=$PLAYGROUND_DEPLOY_USERNAME\necho password=$PLAYGROUND_DEPLOY_PASSWORD' >> /home/circleci/helper-script.sh
       - run:
-          name: Set up git config
-          command: git config credential.helper "/bin/bash /home/circleci/helper-script.sh"
+	  name: Set up git config
+	  command: git config credential.helper "/bin/bash /home/circleci/helper-script.sh"
       - run:
-          name: Deploy
-          command: |
-            echo $PLOTLY_GA_CODE > ga_code.txt
-            cat ga_code.txt
-            pwd
-            APPS_MODIFIED=$(git diff origin/master origin/$CIRCLE_BRANCH --dirstat=files,1 apps/ | awk '{ split($2,a,"/"); if (length(a[2]) != 0) { print a[2]} } ' | sort -u)
-            if [ -z "$APPS_MODIFIED" ]
-            then
-                  echo "No app change detected. Skipping the deploy.."
-                  exit 0
-            fi
+	  name: Deploy
+	  command: |
+	    APPS_MODIFIED=$(git diff origin/master origin/$CIRCLE_BRANCH --dirstat=files,1 apps/ | awk '{ split($2,a,"/"); if (length(a[2]) != 0) { print a[2]} } ' | sort -u)
+	    if [ -z "$APPS_MODIFIED" ]
+	    then
+		  echo "No app change detected. Skipping the deploy.."
+		  exit 0
+	    fi
 
```

Comment:
This whole commit seems to be doing opposite of what you mentioned in the commit message (ie, "change tabs to spaces"). You might want to remove this commit or clean it up.